Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -40,6 +40,15 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
     }()
     
     let newPayPalCheckoutToggle = UISwitch()
+    
+    lazy var rbaDataToggleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Recurring Billing (RBA) Data"
+        label.font = .preferredFont(forTextStyle: .footnote)
+        return label
+    }()
+    
+    let rbaDataToggle = UISwitch()
 
     override func viewDidLoad() {
         super.heightConstraint = 300
```

Comment:
Looks like the view currently is a bit squished, if we update this it should give the checkout view a bit more space:
```suggestion
        super.heightConstraint = 350
```